### PR TITLE
CAB-4314: Adds ability to use x-audituser header for authentication and require it for docfinity routes.

### PR DESCRIPTION
**Pull request by farmas**

### CODE REVIEW

File: src/main/java/edu/uw/edm/edmzuulproxy/config/web/AuthenticationConfiguration.java [hunk 1]
```diff
@@ -46,6 +46,7 @@ public void addCorsMappings(CorsRegistry registry) {
                         .allowedOrigins("*")
                         .allowedMethods("*")
                         .allowedHeaders(securityProperties.getAuthenticationHeaderName())
+                        .allowedHeaders(securityProperties.getDocfinityAuthenticationHeaderName())
```
Comment (farmas) [inline src/main/java/edu/uw/edm/edmzuulproxy/config/web/AuthenticationConfiguration.java:49]: Not 100% sure if this is needed, but I added it here to match the other header.

**Review by agagne — DISMISSED** [review]
Thanks for including the //arange, //act, //assert it is useful to drill in that pattern

**Review by jfshenuw — COMMENTED** [review]
Is x-uw-act-as header required? In other words, would the client have to set both x-uw-act-as and x-audituser headers to the same value?

**Comment by farmas** [comment]
x-uw-act-as is not required for requests to /docfinity, only x-audituser.

**Pull request merged**